Bottom: e57127e4613333957b0bbfff4933af2a5944078b
Top:    9995cf420d2d4e2ce0da2764f0d3adc0f271c2cc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-06 16:33:31 +0000

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 2280665..3d4038e 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1658,15 +1658,15 @@ Gather (VNODE *startv, PLINE **result, J_Rule j_rule, DIRECTION initdir)
   assert (*result == NULL);
   do
     {
-      VNODE *jump_cur_temp;
+      VNODE *jump_curv_temp;
       DIRECTION jump_dir_temp;
 
-      jump_cur_temp = cur;
+      jump_curv_temp = curv;
       jump_dir_temp = dir;
 
 #if 1
       /* see where to go next */
-      if (!jump (&jump_cur_temp, &jump_dir_temp, v_rule))
+      if (!jump (&jump_curv_temp, &jump_dir_temp, j_rule))
 	break;
 #endif
 
@@ -1700,12 +1700,12 @@ Gather (VNODE *startv, PLINE **result, J_Rule j_rule, DIRECTION initdir)
       DEBUGP ("gather vertex at %mm, %mm, Dir=%i\n", curv->point[0], curv->point[1], dir);
 #endif
 #if 1
-      cur = jump_cur_temp;
+      curv = jump_curv_temp;
       dir = jump_dir_temp;
 #endif
 #if 0
       /* see where to go next */
-      if (!jump (&cur, &dir, v_rule))
+      if (!jump (&cur, &dir, j_rule))
 	break;
 #endif
 
@@ -1720,7 +1720,7 @@ Gather (VNODE *startv, PLINE **result, J_Rule j_rule, DIRECTION initdir)
       curv = (dir == FORW) ? NEXT_VERTEX (curv) : PREV_VERTEX (curv);
 #if 0
       /* see where to go next */
-      if (!jump (&cur, &dir, v_rule))
+      if (!jump (&cur, &dir, j_rule))
 	break;
 #endif
     }
